exonum-java-binding-cryptocurrency-demo/frontend/src/plugins/blockchain.js

```diff
@@ -129,35 +112,49 @@ module.exports = {
          message_id: TX_WALLET_ID,
          signature: signature,
          body: data
-        })
+        }).then(response => waitForAcceptance(keyPair.publicKey, response.data))
       },
 
       transfer(keyPair, receiver, amountToTransfer, seed) {
-        
+        // define schema for transaction with protobuf
```

Comment:
I'd consider splitting this and the previous functions into smaller ones to ease reading, and avoid code duplication, as they've become rather long. E.g.:
```
def transferTx:
  createTransferTx(<parameters>)  
  submitTransaction(id, body)
```